docs/source/examples/learning-depolarizing-noise.myst

```diff
@@ -0,0 +1,208 @@
+    circuit=circuit,
+    num_training_circuits=5,
+    fraction_non_clifford=0.2,
+    random_state=np.random.RandomState(1),
+)
+ideal_values = np.array(
+    [ideal_executor.evaluate(tc, observable) for tc in training_circuits]
+)
+
+epsilons = np.linspace(0, 0.1, 21)
+
+import warnings
+
+# suppress warnings about missing representations
+# this example only simulates noise on CNOT
+warnings.simplefilter("ignore", UserWarning)
+loss = [
+    biased_noise_loss_function(
+        [eps, 0],
+        operations_to_learn,
+        training_circuits,
+        ideal_values,
+        noisy_executor,
+        pec_kwargs,
+        observable,
+    )
+    for eps in epsilons
+]
+
+_ = plt.plot(epsilons, loss)
+plt.title("Depolarizing noise loss function")
+plt.xlabel("Noise strength")
+plt.ylabel("Loss")
+plt.show()
+```
+
+Now we set the initial conditions for the optimization. For purposes of this demonstration, our initial guess for epsilon, `epsilon0`, is slightly offset from the true value of `epsilon`.
+
+```{code-cell} ipython3
+offset = 0.1
+epsilon0 = (1 - offset) * epsilon
+```
+
+Use Nelder-Mead method from [`scipy.optimize.minimize`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.minimize.html) to find the optimized value of epsilon.
+
+```{code-cell} ipython3
+[success, epsilon_opt] = learn_depolarizing_noise_parameter(
+    operations_to_learn=operations_to_learn,
+    circuit=circuit,
+    ideal_executor=ideal_executor,
+    noisy_executor=noisy_executor,
+    pec_kwargs=pec_kwargs,
+    num_training_circuits=5,
+    fraction_non_clifford=0.2,
+    training_random_state=np.random.RandomState(1),
+    epsilon0=epsilon0,
+    observable=observable,
+    method="Nelder-Mead",
+)
+
+print(success)
+print(abs(epsilon_opt - epsilon))
+print(offset * epsilon)
+```
+
+Optimization completed successfully and the optimized value of epsilon is closer to the true value of epsilon as compared to the initial value. 
+
+Now we will use the optimized value of the noise strength to map CNOT into its quasiprobability representation.
+
+```{code-cell} ipython3
+representations = [
+    represent_operation_with_local_depolarizing_noise(Circuit(op), epsilon_opt)
+]
+```
+
+Apply PEC on original noisy circuit to obtain the error-mitigated expectation value.
+
+```{code-cell} ipython3
+mitigated = execute_with_pec(
+    circuit=circuit,
+    executor=noisy_executor,
+    observable=observable,
+    representations=representations,
+    **pec_kwargs,
+)
+print(mitigated.real)
+```
+
+Comparing with unmitigated and ideal values, we see that the mitigated value, calculated with the depolarizing noise strength learned from Clifford circuit data is closer to the ideal value, as compared with the noisy value.
+
+```{code-cell} ipython3
+unmitigated = noisy_executor.evaluate(circuit, observable)[0].real
+ideal = ideal_executor.evaluate(circuit, observable)[0].real
+print(unmitigated)
+print(ideal)
+```
```

Comment:
We should be more explicit here about relative errors. It's great for the final line to show that using this technique decreases your error!

Reply:
Agreed. Updated to be relative, but I need to fix the formatting.